doc/dsmdoc.rst

```diff
@@ -0,0 +1,244 @@
+	
+The modelled timesplan lasted also 7 days with five parameters from DSM sheet in 
+``greenland north.xlsx``.  The electricity supply of ``North`` was dominated by 
+wind plants. The wind plants worked for 24 hours in one whole day, and the wind 
+power strong or weak has nothing to do with the change of time. so the parameter 
+``delay`` was set to 8 hours. Because the peak value of the output of ``North`` 
+was just close to 15 GW, the ``cap-max-do`` and ``cap-max-up`` were set to 500 MW, 
+which was a quarter of South's. The electricity in the first four days, which was 
+generated by wind plants, kept at a higher level. That is why the up/downshifts 
+appeared frequently, regularly, and alternately during this time. But in the last 
+three days the wind became weakly, and the electricity of storage had to be taken 
+out to meet the demand. Then during the three days downshifts dominated in most 
+case. With DSM up/downshifts intelligent allocation of electricity resources is 
+required to avoid the shortage of electricity supply during peak hours and the 
+overcapacity in the usual time. 
+
+
+.. image:: greenland/scenario_all_together-Elec-North-sum.png
+    :width: 90%
+    :align: center
+
+.. note::
+
+    For trial e.g. of the result of lower Demand Side Management this
+    :download:`greenland north.xlsx <greenland/greenland north.xlsx>`
+    is the input file used for this scenario
+
+
+
+No Maximal Up/Downshift Capacity 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+All process, transmission and storage capacities are predetermined and constant.
+
+The last scenario illustrates the energy balance of the ``Mid`` of ``greenland``. 
+It has a demand of 50-70 GW that is mostly supplied by a 50 GW transmission, which 
+came from ``South``. In addition, a 13 GW wind plant and 16 GW Photovoltaics plant 
+had made a contribution to the whole electricity system of ``Mid``.
+ 
+
+.. csv-table:: Scenario All Together: Elec in Mid
+    :header-rows: 1
+    :stub-columns: 1
+
+    Process,     eff, inst-cap, inst-cap-out, fuel-cost, var-cost, total-var-cost
+  Photovoltaics, 1.00,  15000,     16000,          0,         0,            0 
+    Wind plant,	 1.00,      0,     13000,          0,         0,            0
+    Gas plant,   0.60,      0,      8000,         27,      1.60,        28.60
+   Hydro plant,  1.00,      0,      1400,          6,       1.40,        7.40
+ Lignite plant,  0.40,      0,     60000,          0,       0.60,        0.60
+ Biomass plant,  0.35,      0,      5000,          6,       1.40,        7.40
+ 	
+
+.. csv-table::DSM in Mid
+    :header-rows: 1
+    :stub-columns: 1
+	
+	Site,   Commodity, delay,  eff,  recov, cap-max-do, cap-max-up
+	 Mid,     Elec,      0,    1.00    1,        0          0
+
+
+Because of the adequate power supply in ``Mid`` the parameters ``delay``, ``cap-max-do``
+and ``cap-max-up`` were set to 0. It means that there was no more DSM in the electricity
+system of ``Mid`` to constraint the commodities.

```

Comment:
I have not cross-checked with the images, but the description seems to be very sparse/short. Could you lengthen these paragraphs a bit (i.e. at least one more sentence)?